packages/generate-scaffold/index.ts

```diff
@@ -0,0 +1,96 @@
+import * as c from "ansi-colors";
+import chalk from "chalk";
+import { exec } from "child_process";
+import * as fs from "fs";
+import * as fse from "fs-extra";
+import * as inquirer from "inquirer";
+import { resolve } from "path";
+
+const cl = console;
```

Comment:
No need to do this

Reply:
@evenstensberg you are mentioning about `const cl = console;`?  If so then if I am directly writing `console.log`  tslint will give the error: no console.log allowed

Reply:
The rule is there for a reason, it means that the logs are not accepted

Reply:
@ematipico yeah! but I really need to print something on the console to make it intractable. Is there any other workaround for this ?

Reply:
pushed with the same for now!